wcfsetup/setup/db/install.sql

```diff
@@ -1446,6 +1446,32 @@ CREATE TABLE wcf1_trophy(
 	KEY(categoryID)
 );
 
+DROP TABLE IF EXISTS wcf1_unfurl_url;
+CREATE TABLE wcf1_unfurl_url (
+	urlID INT(10) NOT NULL AUTO_INCREMENT PRIMARY KEY,
+	url TEXT NOT NULL,
+	urlHash VARCHAR(40) NOT NULL,
+	title VARCHAR(255) NOT NULL DEFAULT '',
+	description TEXT,
+	imageID INT(10),
+	status VARCHAR(255) NOT NULL DEFAULT 'PENDING',
+	lastFetch INT(10) NOT NULL DEFAULT 0,
+
+	UNIQUE KEY urlHash (urlHash)
+);
+
+DROP TABLE IF EXISTS wcf1_unfurl_url_image;
+CREATE TABLE wcf1_unfurl_url_image (
+	imageID INT(10) NOT NULL AUTO_INCREMENT PRIMARY KEY,
+	imageUrl TEXT NOT NULL,
+	imageHash VARCHAR(40) DEFAULT NULL,
+	width INT(10) NOT NULL,
+	height INT(10) NOT NULL,
+	imageExtension VARCHAR(4) DEFAULT NULL,
```

Comment:
```suggestion
	imageExtension VARCHAR(255) DEFAULT NULL,
```
You are not saving space with just 4 characters and it unnecessarily restricts future extensions.

Reply:
I won't change it, because it is technically correct. The intention is *not*  to save space, it is to validate incorrect data is inserted there. Yes, theoretically, there are file extensions with more then 4 characters, but we don't use them. We only use `jpg`, `gif`, `png` and, here and there, `webp`. If we want to use a file extension named `.coolExtension`, then we must adjust this, but I don't see any extension like this in the near future. 

AFAIK we use the `VARCHAR(4)` (or `VARCHAR(3)`) for the file extension on many other places, too. Yes, this is not a real argument, but if we want to support other file extensions, we must adjust these places, too.